templates/zerver/help/change-the-date-and-time-format.md

```diff
@@ -1,21 +1,11 @@
 # Change the date and time format
-![Display time in 24-hour format](/static/images/help/24-hour.png)
+By default, messages in Zulip are displayed with a 12-hour timestamp. If you are used to seeing time displayed in a 24-hour format, you can change the Zulip timestamp format from a 12-hour format to a 24-hour format.
 
-By default, messages in Zulip are displayed with a 12-hour timestamp. If you are used to seeing time displayed in a 24-hour format, you can change the Zulip timestamp format from a 12-hour format to a 24-hour format. You may want to do this to avoid confusion between A.M. and P.M. hours.
-
-You can easily change the date and time format your messages are displayed in by following a few steps.
+Change the date and time format by:
 
 1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner of the right sidebar.
```

Comment:
You'll need to replace steps 1-3 with the **Settings** macro (check the new documentation style guide for reference)